src/SDK/SmartDetectorsSDK/AutomaticResolutionParameters.cs

```diff
@@ -0,0 +1,22 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="AutomaticResolutionParameters.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors
+{
+    using System;
+
+    /// <summary>
+    /// Contains parameters used to handle alerts' automatic resolution
+    /// </summary>
+    public class AutomaticResolutionParameters
+    {
+        /// <summary>
+        /// Gets or sets the duration after which the Alert will be checked for automatic
+        /// resolution by querying the Runtime Environment.
```

Comment:
Consider adding explanation of what method will be called and what interface needs to be implemented. I think many users will discover automatic resolution by looking at this class.

Reply:
added the comment on the AutomaticResolutionParameters property in the Alert class

---
In reply to: [234996878](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/27#discussion_r234996878) [](ancestors = 234996878)